config/routes.rb

```diff
@@ -11,6 +11,9 @@
     post 'installed_packages_inclusion', to: 'uploads#installed_packages_inclusion'
     post 'ips_obfuscation', to: 'uploads#ips_obfuscation'
 
+    get 'cloud_connector', to: 'uploads#cloud_connector_status'
+    post 'cloud_connector', to: 'uploads#enable_cloud_connector'
```

Comment:
@MariaAga I updated the routes here. The POST route should kick things off and return a task. The GET should return the most recent job.